packages/dashboard/src/components/DashboardFrame/Navigation.tsx

```diff
@@ -91,7 +91,7 @@ const LogoItem = styled(MuiListItem)(({ theme }) => ({
 })) as any as typeof MuiListItem
 
 const ItemIcon = styled(ListItemIcon)(({ theme }) => ({
-    [`& svg`]: {
```

Comment:
This is good. I like it. BTW, does the rule provide auto-fixing too?

Reply:
ye, https://github.com/DimensionDev/eslint-plugin/blob/master/src/rules/no-simple-template-literal.ts